index.js

```diff
@@ -90,7 +103,7 @@ const ignored = [
   {
     property: "position",
     value: "static",
-    ignoredProperties: ["top", "right", "bottom", "left"]
+    ignoredProperties: ["top", "right", "bottom", "left", "z-index"]
```

Comment:
> z-index has no effect on this element since its position is static

https://github.com/captainbrosset/useless-css-properties/blob/master/rules.md#misc